models/pardot/model.sql

```diff
@@ -36,13 +36,13 @@ create or replace view pardot_analysis.visitoractivity_types_meta as (
   select 31, 'UserVoice Suggestion' union all
   select 32, 'UserVoice Comment' union all
   select 33, 'UserVoice Ticket' union all
-  select 34, 'Video Watched (≥ 75% watched)'
+  select 34, 'Video Watched (>= 75% watched)'
```

Comment:
@cmerrick totally reasonable that this should be a table for performance reasons. One benefit of it being a view, though, is that end-users can't change the definition w/o writing SQL. We probably don't want people editing the table in their GUI b/c information is lost if the table is wiped. It needs to somehow be specified read-only IMO